### PR TITLE
Add concept ID to tracking events

### PR DESCRIPTION
**Pull request by GlynnPhillips**
Add concept ID to the following tracking events

1. Preference Modal view events
2. CTA click on instant alerts checkbox
3. CTA click on contact preferences link
4. CTA click on remove concept

https://financialtimes.atlassian.net/browse/PT-295

### CODE REVIEW

File: components/jsx/preferences-modal/preferences-modal.jsx [hunk 1]
```diff
@@ -37,6 +36,7 @@ export default function InstantAlertsPreferencesModal({ flags, conceptId, visibl
 							name="receive-instant-alerts"
 							value="receive-instant-alerts"
 							data-component-id="myft-preferences-modal-checkbox"
+							data-trackable-context-concept_id={conceptId}
```
Comment (asugar13) [inline components/jsx/preferences-modal/preferences-modal.jsx:39]: praise/surprise: nice to see this works!

File: components/jsx/preferences-modal/index.js [hunk 2]
```diff
@@ -46,25 +45,34 @@ const positionModal = ({ event, preferencesModal } = {}) => {
...
+			id: '72de123e-5082-11ee-be56-0242ac120002',
+		}
+	}
+
+	const trackingEvent = new CustomEvent('oTracking.event', {
+		detail: trackingData,
+		bubbles: true
+	});
+
+	document.body.dispatchEvent(trackingEvent);
```
Comment (asugar13) [inline components/jsx/preferences-modal/index.js:65]: I've tested this and can see it's working as expected. But out of curiosity, how does the `oTracking.event` work? Is the handling code for this in next-article?

File: components/jsx/preferences-modal/preferences-modal.jsx (hunk 1 above)
Comment (GlynnPhillips) [reply components/jsx/preferences-modal/preferences-modal.jsx:39]: o-tracking readme https://github.com/Financial-Times/next-myft-page/blob/main/client/components/alerts/js/update-preferences.js#L25

File: components/jsx/preferences-modal/index.js (hunk 2 above)
Comment (GlynnPhillips) [reply components/jsx/preferences-modal/index.js:65]: Yes this relies on the consuming app having otracking setup which then listens for these events

**Review by asugar13 — APPROVED** [review]
awesome stuff

**Pull request merged**